src/Razor/Microsoft.AspNetCore.Razor.Tools/src/GenerateCommand.cs

```diff
@@ -25,6 +26,8 @@ public GenerateCommand(Application parent)
             Outputs = Option("-o", "Generated output file path", CommandOptionType.MultipleValue);
             RelativePaths = Option("-r", "Relative path", CommandOptionType.MultipleValue);
             FileKinds = Option("-k", "File kind", CommandOptionType.MultipleValue);
+            CssScopeSources = Option("-cssscopeinput", "CSS scope source file", CommandOptionType.MultipleValue);
```

Comment:
`cssscopeinput` makes it sound like this is the CSS file. Ditto with the description.

Reply:
Good point. Changed `cssscopeinput` to `cssscopedinput` (to clarify that the scoping is something happening to the input, rather than the input being the scope), and updated descriptions. The term `input` here is still applicable because of its correspondence to the other parameter named `input`.